src/DotNetWorker/Definition/FunctionMetadata.cs

```diff
@@ -12,5 +14,7 @@ public abstract class FunctionMetadata
         public abstract string FunctionId { get; set; }
 
         public abstract string Name { get; set; }
+
+        public abstract IEnumerable<string> BindingNames { get; set; }
```

Comment:
We get `IDictionary<string, BindingMetadata>` from grpc. But for now, only need the strings.